ccd-definition/CaseEventToComplexTypes/manageHearings/manageHearings.json

```diff
@@ -114,7 +114,6 @@
     "CaseEventID": "manageHearings",
     "CaseFieldID": "previousHearingVenue",
     "ListElementCode": "newVenue",
-    "EventElementLabel": "Venue",
```

Comment:
This was overwriting venue with venue, bit redundant, so I removed it and at UX request we have changed it to be Court instead of venue as they thought people could be confused by the wording